pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.poetry]
+name = "securedrop-docs"
+version = "2.5.2"
+description = "SecureDrop documentation for journalists, sources and administrators"
+authors = ["SecureDrop team <securedrop@freedom.press>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+sphinx = "^6.1.3"
```

Comment:
In redmine_docs, you had:
```
# The RTD theme does not yet support Sphinx >=6.0.0.
# See https://github.com/readthedocs/sphinx_rtd_theme/issues/1323
sphinx = "<6.0.0"
```

Is that not necessary in this repo?

Reply:
I didn't hit any compatibility issues with the most recent versions. I'll see if those have been resolved and we can update `redmine_docs` as well.